elasticsearch/example/src/test/java/example/springdata/elasticsearch/conference/ElasticsearchAvailable.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package example.springdata.elasticsearch.conference;
+
+import static org.junit.platform.commons.support.AnnotationSupport.findAnnotation;
+
+import java.io.IOException;
+import java.util.Optional;
+
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpHead;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.junit.jupiter.api.extension.ConditionEvaluationResult;
+import org.junit.jupiter.api.extension.ExecutionCondition;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+/**
+ * @author Christoph Strobl
+ * @author Prakhar Gupta
+ */
+public class ElasticsearchAvailable implements ExecutionCondition {
```

Comment:
It would make sense to refactor this condition into a utility module so we don't need to repeat it all over.

Reply:
Already done as part of https://github.com/spring-projects/spring-data-examples/commit/b5cecf730bf8e034c2dfc8eb84695cdd331a6202